### ISSUE TITLE
照抄的代码，结果不对，是我哪里遗漏了

### ISSUE THREAD

**Issue by RockNHawk** [nghuyong/ERNIE-Pytorch#68]
## 输出
是不是和下面第一句提示有关系？

Some weights of the model checkpoint at nghuyong/ernie-1.0-base-zh were not used when initializing ErnieModel: ['cls.predictions.transform.dense.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.bias', 'cls.predictions.transform.dense.bias']
- This IS expected if you are initializing ErnieModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing ErnieModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
huggingface result
predict result:  ['由', '之', '络', '儿', '之', '版', '料', '并', '之', '任', '，', '之', '细', '存', '细', '只', '求', '王', '之', '之', '之', '立', '英', '王', '求', '之', '立', '英', '王', '之', '之', '立', '龙', '日', '看', '求', '之', '情', '很', '很', '照', '评', '生', '群', '状']
[CLS] logit:     [ 8.25601876e-01 -2.07599640e-01 -4.17745471e-01  4.74510729e-01

# 完整代码

	from transformers import BertTokenizer, ErnieModel
	import torch
	import numpy

	tokenizer = BertTokenizer.from_pretrained("nghuyong/ernie-1.0-base-zh")
	model = ErnieModel.from_pretrained("nghuyong/ernie-1.0-base-zh")

	input_ids = torch.tensor([tokenizer.encode(text="[MASK][MASK][MASK]是中国神魔小说的经典之作，与《三国演义》《水浒传》《红楼梦》并称为中国古典四大名著。",
												   add_special_tokens=True)])
	model.eval()
	with torch.no_grad():
		predictions = model(input_ids)[0][0]
	predicted_index = [torch.argmax(predictions[i]).item() for i in range(predictions.shape[0])]
	predicted_token = [tokenizer._convert_id_to_token(predicted_index[i]) for i in
	range(1, (predictions.shape[0] - 1))]
	print('huggingface result')
	print('predict result:\t', predicted_token)
	print('[CLS] logit:\t', predictions[0].numpy())

**Comment by nghuyong** [comment]
https://github.com/nghuyong/ERNIE-Pytorch/blob/master/test.py#L22

复现这个例子是 ErnieForMaskedLM 而不是 ErnieModel

**Comment by RockNHawk** [comment]
> ErnieForMaskedLM

哇，确实可以了。感谢到老深夜指点。刷了一下居然真的刷到大佬的回复，非常非常感谢。
